### ISSUE TITLE
freeswitch-license-server segfault after update to fs 10.10 and debian 12

### ISSUE THREAD

**Issue by ynasida** [signalwire/freeswitch#2272]
this happens immediately after trying to start

# freeswitch-license-server

fs kernel: [72774.127317] freeswitch-lice[118577]: segfault at 27040 ip 0000000000027040 sp 00007ffc16e89098 error 14 in freeswitch-license-server[400000+1000]
fs kernel: [72774.127330] Code: Unable to access opcode bytes at RIP 0x27016.

**Comment by andywolk** [comment]
What mod are you using with this?

**Comment by ynasida** [comment]
> What mod are you using with this?

I did
apt-get install freeswitch-license-server
apt-get install freeswitch-mod-com-g729

**Comment by andywolk** [comment]
Thank you. Let me see what I can do.

**Comment by andywolk** [comment]
Please try again.
Just make sure you don't have your FreeSWITCH running while doing this. I suggest you to apt update and install the latest:
```
apt-get update
apt-get install freeswitch-license-server
apt-get install freeswitch-mod-com-g729
```

**Comment by ynasida** [comment]
Very sorry, I just noted that I had updated bullseye freeswitch packets installed at Debian 12 bookworm. It was fine for Freeswitch (for some reason) but probably was not good for freeswitch-license-server. 

I had them installed:
freeswitch-license-server is already the newest version (0.0.1~6~d6de7efa7a~bullseye).
freeswitch-mod-com-g729 is already the newest version (1.10.7~release~31~5a238d3766~bullseye).

I did changes to freeswitch.list (set bookworm) and re-installed all FS packets. It works fine now.

May be your patch also affected - I do not know.

Sorry again.
